src/components/Table/SearchTable.tsx

```diff
@@ -1,46 +1,60 @@
+import { useEffect, useRef } from 'react';
+import exampleData from './example-data.json';
+
 interface SearchTableProps {
   cancelEvent?: boolean;
 }
 
 const SearchTable = ({ cancelEvent }: SearchTableProps) => {
+  const searchTable = useRef<HTMLTdsTableElement>(null);
```

Comment:
Looks like both the searchable Tables are filtered when searching in the first Table:
![image](https://github.com/scania-digital-design-system/tegel-react-demo/assets/48509966/c19e3dbe-27d3-4cc2-bf57-a040fa4320cb)

Reply:
thought: they might shade an ID or something like that

Reply:
I have rebased the branch and now the issues are gone. Will keep my eyes while doing future work.